pkg/spiffs/doc.txt

```diff
@@ -1,7 +1,6 @@
 /**
  * @defgroup pkg_spiffs   SPI flash file system
  * @ingroup  pkg
- * @ingroup  sys_fs
```

Comment:
Mhhh. Why remove this line? @aabadie introduced it intentionally in https://github.com/RIOT-OS/RIOT/pull/7298